### PR TITLE
Feature/hex earnings

### PR DESCRIPTION
**Pull request by matthewcarlreetz**

### CODE REVIEW

File: components/Map/MapControls.js [hunk 1]
```diff
@@ -72,6 +73,43 @@ const MapControls = () => {
...
+          },
+        )}
+      >
+        <div className="rounded-lg flex flex-row items-center justify-between titlebox-blur">
+          <a
+            href="https://docs.helium.com/wallets/app-wallet/hexagons/#transmit-scale"
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            <Tooltip title="Rocket good, zero bad">
```
Comment (matthewcarlreetz) [inline components/Map/MapControls.js]: @cokes518 Tooltip text is here.

File: components/Map/MapControls.js [hunk 2]
```diff
@@ -72,6 +73,43 @@ const MapControls = () => {
...
+            'opacity-100 pointer-events-auto':
+              mapLayer === 'earnings' && !showMapLayers,
+            'opacity-0 pointer-events-none translate-y-10':
+              mapLayer !== 'earnings' || showMapLayers,
+          },
+        )}
+      >
+        <div className="rounded-lg flex flex-row items-center justify-between titlebox-blur">
+          <a
+            href="https://docs.helium.com/wallets/app-wallet/hexagons/#transmit-scale"
```
Comment (matthewcarlreetz) [inline components/Map/MapControls.js]: Need to update or remove this href @cokes518

Comment (cokes518) [reply components/Map/MapControls.js]: removed the href until we add it to the app, then I'll update the wallet hexagon guide to include the earnings map layer.

**Review by danielcolinjames — APPROVED** [review]
looks good! super useful feature 👍

**Pull request merged**